.github/workflows/playwright.yml

```diff
@@ -0,0 +1,78 @@
+name: Playwright E2E Tests
+
+on:
+  push:
+    branches:
+      - "develop"
+      # - "feature/**"
```

Comment:
should we just delete this? I think I copied and pasted this from `js-tests.yml` which I assume you did too and I just left it there.